requirements.txt

```diff
@@ -13,8 +13,10 @@ matplotlib==3.1.1
 numpy==1.16.4
 opt-einsum==3.1.0
 protobuf==3.10.0
+pydub==0.23.1
 pyparsing==2.4.5
 python-dateutil==2.8.1
+scipy==1.3.2
 six==1.13.0
 tb-nightly==1.15.0a20190806
```

Comment:
Why are we using the nightly build of tensorboard? Is it possible to use a stable build?

Reply:
If not it's fine, just want to make sure this is really necessary.